pyof/foundation/network_types.py

```diff
@@ -138,6 +139,135 @@ def get_size(self, value=None):
         return 2 + self.length
 
 
+class IPv4(GenericStruct):
+    """IPv4 packet "struct".
+
+    Contains all fields of an IP version 4 packet header, plus the upper layer
+    content as binary data.
+    Some of the fields were merged together because of their size being
+    inferior to 8 bits.
+    """
+
+    # IP protocol version + Internet Header Length (words)
```

Comment:
@renanrodrigo for these comments work as documentation, you must use `#:` instead of only `#`.